sample-dcrstakepool.conf

```diff
@@ -88,6 +88,8 @@ testnet=1
 ; Specified extended public key is used to generate ticketed addresses
 ; which are combined with a user address for 1-of-2 multisig.
 ; Must be the voting wallet's masterpubkey for the default account.
+; DEPRECATED: This option has no effect. votingwalletextpub is derived
```

Comment:
Here again the word "derived" is a bit confusing.  It is already derived in the wallet, dcrstakepool just retrieves it.